deepchem/models/torch_models/tests/test_prot_bert.py

```diff
@@ -0,0 +1,136 @@
+import os
+
+import deepchem as dc
+import pytest
+
+try:
+    import torch
+    from deepchem.models.torch_models.prot_bert import ProtBERT
+except ModuleNotFoundError:
+    pass
+
+
+@pytest.mark.torch
+def test_prot_bert_pretraining_mlm(protein_classification_dataset):
+    model_path = 'Rostlab/prot_bert'
+    model = ProtBERT(task='mlm', HG_model_path=model_path, n_tasks=1)
+    loss = model.fit(protein_classification_dataset, nb_epoch=1)
+    assert loss
```

Comment:
What is the range of loss values you are getting?

Reply:
Around 0.4